### PR TITLE
Fixes trivial lint suggestions from default clippy

### PR DESCRIPTION
**Pull request by joelgallant**

### CODE REVIEW

File: src/v4l2.rs [hunk 1]
```diff
@@ -640,7 +641,7 @@ pub mod pubconsts {
     pub const CID_USER_ADV7180_BASE: u32 = CID_USER_BASE + 0x1070;
     pub const CID_MPEG_BASE: u32 = CLASS_MPEG | 0x900;
     pub const CID_MPEG_CLASS: u32 = CLASS_MPEG | 1;
-    pub const CID_MPEG_STREAM_TYPE: u32 = CID_MPEG_BASE+0;
```
Comment (loyd) [inline src/v4l2.rs]: I use `+ 0` here and below to emphasize a difference between `_BASE` and some control.

Comment (joelgallant) [reply src/v4l2.rs]: I figured that was the case, though from my perspective I don't think `+ 0` adds much clarity

**Comment by joelgallant** [comment]
Rebased on latest

**Comment by loyd** [comment]
Thanks!

**Pull request merged**